### PR TITLE
Feat/36 blogs page

### PR DESCRIPTION
**Pull request by Cameron933**
### Descriptions
1. refactored MemberCard component.
2. completed blog page with testing

- Link to Vercel preview: https://website-git-feat-36-blogs-page-taodemy.vercel.app/blog
- Link to Zenhub: https://app.zenhub.com/workspaces/taodemy-63c3dec267d0df73ca6087cb/issues/gh/taodemy/website/36
- Link to UI design (if possible): https://www.figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=0%3A1&t=kWkf0CRDhQoT32Nc-0

### Provide screenshots or videos that the function is working.
image
image
image

### Provide screenshots of no warning, or error in the console.
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by Trent1900 — COMMENTED** [review]
LGTM

**Review by HarryuCATGREY — APPROVED** [review]
LGTM

File: components/general/Blog/BlogGallery.module.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  padding-bottom: 63px;
+  display: flex;
+  gap: 48px;
+  border-bottom: var(--default-border);
+}
+
+.blog_gallery__small_group {
+  padding-top: 63px;
+  display: flex;
+  flex-direction: column;
```
Comment (HarryuCATGREY) [inline components/general/Blog/BlogGallery.module.css:19]: How can you make it auto-fill to both ends without space-between?

image

File: styles/MemberCard.module.css [hunk 2]
```diff
@@ -46,7 +46,3 @@
 .card__title {
   font-weight: 500;
 }
```
Comment (tatoMa) [inline styles/MemberCard.module.css:48]: Do we need additional font-weight? Is there any pre-defined font style in our global CSS files?

File: pages/blog.tsx [hunk 3]
```diff
@@ -9,6 +11,8 @@ export default function Blog() {
         <meta name="description" content="Generated by Taodemy" />
```
Comment (tatoMa) [inline pages/blog.tsx:11]: Think about SEO again.

**Review by tatoMa — CHANGES_REQUESTED** [review]
All of the files inside of general/Blog, they do not need inside of a general folder. 
Also need to care about naming conventions.

Think about components/blog/Blogxxx.tsx

File: components/general/Blog/BlogGallery.module.css (hunk 1 above)
Comment (Cameron933) [reply components/general/Blog/BlogGallery.module.css:19]: You look into wrong layer, I use the gap between two lines of small img group.

**Comment by Cameron933** [comment]
> All of the files inside of general/Blog, they do not need inside of a general folder. Also need to care about naming conventions.
> 
> Think about components/blog/Blogxxx.tsx

image

File: pages/blog.tsx (hunk 3 above)
Comment (Cameron933) [reply pages/blog.tsx:11]: image

File: styles/MemberCard.module.css (hunk 2 above)
Comment (Cameron933) [reply styles/MemberCard.module.css:48]: We do not need additional font-weight. The font-size & line-height are pre-defined in the global CSS.

**Review by tatoMa — APPROVED** [review]

**Pull request merged**